tests/test_fft.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+import dpnp
+
+import numpy
+
+@pytest.mark.parametrize("type",
+                         [numpy.complex128, numpy.complex64, numpy.float32, numpy.float64, numpy.int32, numpy.int64],

```

Comment:
```suggestion
                         [dpnp.complex128, dpnp.complex64, dpnp.float32, dpnp.float64, dpnp.int32, dpnp.int64],

```

Reply:
In this case this types are foreign for `numpy` in test suit.
It is better use strings for both lib `dtype` constructions.

Reply:
This test is in DPNP test suite. I don't understand what does it mean "foreign".